tests/stress/cases/modules/main.tf

```diff
@@ -0,0 +1,29 @@
+provider "layer0" {
```

Comment:
This Terraform configuration is more-or-less doing the single configuration approach I had mentioned earlier. The difference being the use of sub-modules here. With some interpolation, we can get rid of the sub-modules and combine everything we need into a single configuration.

Reply:
Interesting idea may be to randomly shuffle entities across environments using the [random shuffle](https://www.terraform.io/docs/providers/random/r/shuffle.html) provider

Reply:
`random_shuffle` would be cool if these tests were part of a CI pipeline that was executed on a frequent basis. Not sure if that was our plan overall for these tests, though.

Reply:
We shouldn't be automating these benchmarking/stress tests in a pipeline, I don't think, since they're expensive and lengthy, and don't really add testing coverage that other unit/smoke/system tests do. I think `random_shuffle` is just a way to distribute entities across other entities.

Reply:
💯 🍪

Reply:
Update: A single-file Terraform configuration has been achieved; `random_shuffle`, while cool and fun and interesting, doesn't provide any added value here. I've refactored to use `element()` again, which has modulus built-in.

✅